src/main/java/it/unitn/ds1/Client.java

```diff
@@ -33,20 +38,41 @@ public static Props props(ArrayList<ActorRef> replicas) {
         return Props.create(Client.class, () -> new Client(replicas));
     }
 
-    @Override
-    public void preStart() {
+    /**
+     * Return the replica to be contacted. If favourite replica is available, it
+     * is returned, otherwise another random one is choosen and set as favourite.
+     * @return replica to be contacted
+     */
+    private ActorRef getReplica() {
```

Comment:
I don't know if we should do this. Was it in the description to have a favorite replica? 
Isn't it fine if we choose a random replica every time?

Reply:
I remember reading that they system is sequentially consistent if every client always contact the same replica, but this is just a property and not an actual requirment so yeah we can just choose a random replica every time.